src/Navigation.jsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+import Button from './Button'
+import './Navigation.css'
+
+export default function Navigation({ navButtons, clicked }) {
```

Comment:
Maybe also call the `navButtons` `navText`? Or `navButtonText`? `navButtons` suggests that these already are buttons.